client/src/main/java/org/spine3/protobuf/EntityPackagesMap.java

```diff
@@ -62,7 +62,7 @@
      * <p>Example:
      * <p>{@code com.example.order} - {@code AGGREGATE}
```

Comment:
People may want to have multiple aggregates per package. It would be safer to map from message type to an entity type.